autotelic/fastify-injector
# feat: update deps & fix fastify warnings

# Summary

- Update dependencies
- Replace lint tool -  `@autotelic/eslint-config-js` instead of `standard`
- Update validate and release workflows
- Fix fastify warnings
- Update tests
- Update node version

# Test Plan

- Run tests - there should not be any fastify warnings
- To test out a consumer, run tests for any project that is using fastify-injector, for example [fastify-openapi-autoload](https://github.com/autotelic/fastify-openapi-autoload). Confirm the async/await warnings surface.
- Switch the version of fastify-injector from `"0.2.0"` to ` "github:autotelic/fastify-injector#feat/update"` and install
- Run the tests again and confirm warnings are no longer present
